src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -50,9 +50,19 @@ public class FrameRenderer : CardView, IPlatformViewHandler
 		public event EventHandler<VisualElementChangedEventArgs>? ElementChanged;
 		public event EventHandler<PropertyChangedEventArgs>? ElementPropertyChanged;
 
-		public FrameRenderer(Context context) : base(context)
+		public FrameRenderer(Context context) : this(context, Mapper)
 		{
-			_viewHandlerWrapper = new ViewHandlerDelegator<Frame>(Mapper, CommandMapper, this);
+		}
+
+		// TODO NET8 make public
+		internal FrameRenderer(Context context, IPropertyMapper mapper)
```

Comment:
Should our process here be to start these off as public in the PR to main and then when we backport, change to internal?